### PR TITLE
Rn app manager update with ajax

### PR DESCRIPTION
**Pull request by Rohit25negi**
this code enables the ajax form submission for update prompt setting.
Here is the ticket:
http://manage.dimagi.com/default.asp?272410

### CODE REVIEW

File: corehq/apps/app_manager/static/app_manager/js/apps_base.js [hunk 1]
```diff
@@ -15,4 +15,40 @@ hqDefine("app_manager/js/apps_base", function() {
 
         hqImport("hqwebapp/js/layout").setIsAppbuilderResizing(true);
     });
+
+    $("#update-manager").submit(function(event){
+            event.preventDefault();
+
+            const form = $(this),
```
[the remark this answers is not in the document]
Comment (orangejenny) [reply corehq/apps/app_manager/static/app_manager/js/apps_base.js]: yeah, we're ES5 for the foreseeable future

**Review by orangejenny — CHANGES_REQUESTED** [review]
Hi Rohit, could you take a look at implementing this with the SaveButton? It takes care of a bunch of standard functionality (try again logic, disabling the button until changes are made, etc.) The [linked whitelist](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/static/app_manager/js/settings/linked_whitelist.js) is an small area that uses the SaveButton.

**Comment by Rohit25negi** [comment]
@orangejenny  replaced the previous ajax implementation with save Button implementation.
Tested also.

File: corehq/apps/app_manager/templates/app_manager/partials/releases.html [hunk 2]
```diff
@@ -65,8 +65,10 @@
                 {% include 'app_manager/partials/releases_table.html' %}
             </div>
             {% if enable_update_prompts %}
+                 <script src="{% static 'app_manager/js/releases/update_prompt.js' %}"></script>
```
Comment (orangejenny) [inline corehq/apps/app_manager/templates/app_manager/partials/releases.html]: This should go in `app_manager/app_view.html`. We're moving away from using script tags and towards using requirejs, and that migration is a little bit easier if all the script tags are in the `js` blocks in the big base templates.

**Review by orangejenny — APPROVED** [review]
This looks good, thank you for reworking it. Left one minor comment.

**Comment by Rohit25negi** [comment]
@orangejenny   fyi shifted that script import.

**Review by orangejenny — APPROVED** [review]

**Pull request merged**